projects/vdk-core/src/vdk/internal/builtin_plugins/run/cli_run.py

```diff
@@ -120,6 +120,26 @@ def __warn_on_python_version_disparity(
                         """
                     )
 
+    def __log_exec_result(self, execution_result):
```

Comment:
Those methods seem more suitable for some "utils" module.

Reply:
Refactoring is out of scope for this PR. I'm just copying existing code to a new function. We can open a ticket to take a look at this at a later time.